test/controllers/homepage_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+describe HomepageController do
+  # it "must be a real test" do
+  #   flunk "Need real tests"
+  # end
```

Comment:
Looks like this was forgotten. Aside from the nominal case, there's an interesting edge case when there are no works, since the spotlight section might fail to render.